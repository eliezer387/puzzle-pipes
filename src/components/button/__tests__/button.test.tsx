import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Button from '..';

describe('Button', () => {

    it('should render', () => {
        const { getByText } = render(<Button>Button</Button>);
        expect(getByText('Button')).toBeInTheDocument();
    });

    it('match snapshot', () => {
        const { asFragment } = render(<Button>Button</Button>);
        expect(asFragment()).toMatchSnapshot();
    })
})