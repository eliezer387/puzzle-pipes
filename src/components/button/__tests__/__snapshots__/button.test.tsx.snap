// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button match snapshot 1`] = `
<DocumentFragment>
  <button
    class="sc-bdvvtL jAPvEI"
  >
    Button
  </button>
</DocumentFragment>
`;
